test/events.js

```diff
@@ -161,186 +168,202 @@
-    a.listenTo(b, 'all', function(){ ok(true); });
+    a.listenTo(b, 'all', function(){ assert.ok(true); });
     b.trigger('anything');
-    a.listenTo(b, 'other', function(){ ok(false); });
+    a.listenTo(b, 'other', function(){ assert.ok(false); });
     a.stopListening(b, 'other');
     a.stopListening(b, 'all');
-    equal(_.size(a._listeningTo), 0);
+    assert.equal(_.size(a._listeningTo), 0);
   });
 
-  test("listenToOnce without context cleans up references after the event has fired", 2, function() {
+  QUnit.test("listenToOnce without context cleans up references after the event has fired", function(assert) {
+    assert.expect(2);
     var a = _.extend({}, Backbone.Events);
     var b = _.extend({}, Backbone.Events);
-    a.listenToOnce(b, 'all', function(){ ok(true); });
+    a.listenToOnce(b, 'all', function(){ assert.ok(true); });
     b.trigger('anything');
-    equal(_.size(a._listeningTo), 0);
+    assert.equal(_.size(a._listeningTo), 0);
   });
 
-  test("listenToOnce with event maps cleans up references", 2, function() {
+  QUnit.test("listenToOnce with event maps cleans up references", function(assert) {
+    assert.expect(2);
     var a = _.extend({}, Backbone.Events);
     var b = _.extend({}, Backbone.Events);
     a.listenToOnce(b, {
-      one: function() { ok(true); },
-      two: function() { ok(false); }
+      one: function() { assert.ok(true); },
+      two: function() { assert.ok(false); }
     });
     b.trigger('one');
-    equal(_.size(a._listeningTo), 1);
+    assert.equal(_.size(a._listeningTo), 1);
   });
 
-  test("listenToOnce with event maps binds the correct `this`", 1, function() {
+  QUnit.test("listenToOnce with event maps binds the correct `this`", function(assert) {
+    assert.expect(1);
     var a = _.extend({}, Backbone.Events);
     var b = _.extend({}, Backbone.Events);
     a.listenToOnce(b, {
-      one: function() { ok(this === a); },
-      two: function() { ok(false); }
+      one: function() { assert.ok(this === a); },
+      two: function() { assert.ok(false); }
     });
     b.trigger('one');
   });
 
-  test("listenTo with empty callback doesn't throw an error", 1, function(){
+  QUnit.test("listenTo with empty callback doesn't throw an error", function(assert) {
+    assert.expect(1);
     var e = _.extend({}, Backbone.Events);
     e.listenTo(e, "foo", null);
     e.trigger("foo");
-    ok(true);
+    assert.ok(true);
   });
 
-  test("trigger all for each event", 3, function() {
+  QUnit.test("trigger all for each event", function(assert) {
+    assert.expect(3);
     var a, b, obj = { counter: 0 };
     _.extend(obj, Backbone.Events);
-    obj.on('all', function(event) {
-      obj.counter++;
-      if (event == 'a') a = true;
-      if (event == 'b') b = true;
-    })
-    .trigger('a b');
-    ok(a);
-    ok(b);
-    equal(obj.counter, 2);
-  });
-
-  test("on, then unbind all functions", 1, function() {
+    obj
+      .on('all', function(event) {
```

Comment:
I don't think I've seen that style of chained methods before. It might be worth trying to keep the pull request scoped to just the QUnit changes and leave the code formatting for another PR.

That being said, take my advice with a grain of salt, I'm not a real contributor, just and interested user. I opened the tests for the first time today, and I could not agree with you more. The code "style" looks like it could use some love. Usually I'm not one to get too opinionated about code style, but I found the inconsistencies made it very hard to read.